### PR TITLE
connected store added in gift cards feature

### PR DESCRIPTION
**Pull request by Ravulavamsikrishna**
Gift cards:-
1.connected store added,
2.gift card features added

### CODE REVIEW

File: schemas/partners/index.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { FeatureSchemaDefinition } from "schemas/types";
+
+export const partners: FeatureSchemaDefinition = {
+    variants: {
+        group:[],
+        card:[
+            {
+                title:"[partners] Image Carousel",
```
Comment (NarsingRao13) [inline schemas/partners/index.ts:8]: Cann't we use existing variants?

File: schemas/hotel-details/index.ts [hunk 2]
```diff
@@ -95,6 +95,10 @@ export const details: FeatureSchemaDefinition = {
         title: "[Details] Card with Left Align Content",
         value: "details.card.left-align-content",
       },
+      {
+        title: "[Details] Room Specification",
+        value: "details room-specification",
```
Comment (NarsingRao13) [inline schemas/hotel-details/index.ts:100]: Check nomenclature here.
follow  namespace.type.variant

File: schemas/partners/index.ts (hunk 1 above)
Comment (NarsingRao13) [reply schemas/partners/index.ts]: Attach SS.

Comment (Ravulavamsikrishna) [reply schemas/partners/index.ts]: New feature created:partners
Screenshot 2023-03-13 191057

Comment (NarsingRao13) [reply schemas/partners/index.ts]: Attach figma UI ss for the component.

Comment (Ravulavamsikrishna) [reply schemas/partners/index.ts]: image

**Pull request merged**